malwarefront/src/components/AttributeUpdate.js

```diff
@@ -300,64 +302,67 @@ class AttributeUpdate extends Component {
             return this.state[field] !== this.state["original_"+field] ? <span style={{color: "red"}}>*</span> : <span/>
         };
 
+        let success = this.state.succes && <div>Attribute {this.props.match.params.metakey} modified successfully.</div>
+
         return (
-            <div className="container">
-                <ConfirmationModal isOpen={this.state.isModalOpen}
-                                   onRequestClose={() => this.setState({isModalOpen: false})}
-                                   onConfirm={this.state.modalSpec.action}
-                                   message={this.state.modalSpec.message}
-                                   buttonStyle={this.state.modalSpec.buttonStyle}
-                                   confirmText={this.state.modalSpec.confirmText} />
-                <Alert error={this.state.error} />
-                <h2>Modify attribute {this.props.match.params.metakey}</h2>
-                <form onSubmit={this.handleSubmit}>
-                    <div className="form-group">
-                        <label>Key</label>
-                        <input type="text" className="form-control" value={this.props.match.params.metakey} disabled/>
-                        <div class="form-hint">Key must contain only lowercase letters and digits, max 32 characters allowed.</div>
-                    </div>
+            <View error={this.state.error} success={success}>
+                <div className="container">
```

Comment:
This `<div className="container">` should be removed, it's redundant. `<View>` already wraps children in `container`.
Ref: https://github.com/CERT-Polska/malwarecage/blob/master/malwarefront/src/commons/ui/View.js#L9